### ISSUE TITLE
PyPi package is not up to date

### ISSUE THREAD

**Issue by RianFuro** [chardet/chardet#83]
Hi! I just compared your github codebase to the your PyPi package (installed via pip) and noticed that the code there seems to be out of date (even though it gives the same version number).

Would be cool to have an up-to-date version on pypi since there's a problem with ascii escape sequences in the pypi package that doesn't seem to exist with the github version :)

**Comment by sigmavirus24** [comment]
Hi @RianFuro,

We're planning a large release but neither @dan-blanchard nor I have very much time for chardet. It isn't as simple as updating the package on PyPI because we have to pull together the last few pull requests, documentation, changelog entries, and testing before releasing it.

The version number being the same is purely because we do not change the version number here until we're ready to release the version.

**Comment by adamn** [comment]
I would like a release as well.  I got this traceback recently and the offending code looks alot cleaner now so is probably already fixed.

P.S. - I know the below code is embedded in Requests - I'll create a ticket for them to use the PyPI version of chardet once it's released :-)

```python
builtins:MemoryError
........
File "/var/virtualenvs/centr/lib/python3.5/site-packages/requests/packages/chardet/__init__.py", line 30, in detect
File "/var/virtualenvs/centr/lib/python3.5/site-packages/requests/packages/chardet/universaldetector.py", line 128, in feed
File "/var/virtualenvs/centr/lib/python3.5/site-packages/requests/packages/chardet/charsetgroupprober.py", line 64, in feed
File "/var/virtualenvs/centr/lib/python3.5/site-packages/requests/packages/chardet/sbcharsetprober.py", line 72, in feed
File "/var/virtualenvs/centr/lib/python3.5/site-packages/requests/packages/chardet/charsetprober.py", line 57, in filter_without_english_letters
File "/var/virtualenvs/centr/lib/python3.5/re.py", line 182, in sub
```

**Comment by sigmavirus24** [comment]
@adamn please don't create an issue for requests. I'm a core developer in both places. When we have a good version here, it's my intent to update it in requests. Thanks :)

**Comment by dan-blanchard** [comment]
The main thing blocking a new release (for me) is that we still have failing tests. I would like to have them all passing before we push out a release.

There's also a major set of changes in [this branch](https://github.com/chardet/chardet/tree/abandoned_pr) based off #52, an abandoned PR with loads of good changes (including the ability to retrain out models).  I'm going to try to put in some time to finish that up this week.

**Comment by leonsim** [comment]
Any good news? anything I can help? 😄

**Comment by dan-blanchard** [comment]
I've made some process on the branch I was talking about before, but it's still not 100%. The main issue is that I'm working on the script to retrain our language models, since the one in the abandoned PR are not quite right, and neither are the old ones we're using currently. I recently moved, so I've been busy with house stuff, but I'll try to snag some more time for this soon.

**Comment by dan-blanchard** [comment]
I will try to cut a new release today. I know having some of the fixes we have in master would be better than waiting endlessly for the big one (#99) to be done.

**Comment by dan-blanchard** [comment]
Fixed by #110.
